code/_globalvars/lists/reagents.dm

```diff
@@ -1,12 +1,61 @@
\ No newline at end of file
+var/list/drinks = list("beer2","hot_coco","orangejuice","tomatojuice","limejuice","carrotjuice",
+					"berryjuice","poisonberryjuice","watermelonjuice","lemonjuice","banana",
+					"nothing","potato","milk","soymilk","cream","coffee","tea","icecoffee",
+					"icetea","cola","nuka_cola","spacemountainwind","thirteenloko","dr_gibb",
+					"space_up","lemon_lime","beer","whiskey","gin","rum","vodka","holywater",
+					"tequilla","vermouth","wine","tonic","kahlua","cognac","ale","sodawater",
+					"ice","bilk","atomicbomb","threemileisland","goldschlager","patron","gintonic",
+					"cubalibre","whiskeycola","martini","vodkamartini","whiterussian","screwdrivercocktail",
+					"booger","bloodymary","gargleblaster","bravebull","tequillasunrise","toxinsspecial",
+					"beepskysmash","salglu_solution","irishcream","manlydorf","longislandicedtea",
+					"moonshine","b52","irishcoffee","margarita","blackrussian","manhattan",
+					"manhattan_proj","whiskeysoda","antifreeze","barefoot","snowwhite","demonsblood",
+					"vodkatonic","ginfizz","bahama_mama","singulo","sbiten","devilskiss","red_mead",
+					"mead","iced_beer","grog","aloe","andalusia","alliescocktail","soy_latte",
+					"cafe_latte","acidspit","amasec","neurotoxin","hippiesdelight","bananahonk",
+					"silencer","changelingsting","irishcarbomb","syndicatebomb","erikasurprise","driestmartini")
+
+//Random chem blacklist
+var/global/list/blocked_chems = list("polonium", "initropidril", "concentrated_initro",
+							"sodium_thiopental", "ketamine", "coniine",
+							"adminordrazine", "nanites", "hellwater",
+							"mutationtoxin", "amutationtoxin", "venom",
+							"spore", "stimulants", "stimulative_agent",
+							"syndicate_nanites", "ripping_tendrils", "boiling_oil",
+							"envenomed_filaments", "lexorin_jelly", "kinetic",
+							"cryogenic_liquid", "dark_matter", "b_sorium",
+							"reagent", "life")
```

Comment:
For those wondering, "reagent" is the id for the base /datum/reagent type and any later defined subtype that does not define an id. I know of one off-hand, which is the /datum/reagent/blob subtype, which is the basis for all other blob chems. This inclusion was to be extra safe with that base blob chem.